src/solc_0.5/LandV3.sol

```diff
@@ -72,12 +76,14 @@ contract LandV3 is LandBaseTokenV3, OperatorFiltererUpgradeable {
     function register(address subscriptionOrRegistrantToCopy, bool subscribe) external onlyAdmin {
         require(subscriptionOrRegistrantToCopy != address(0),"LandV3: subscription can't be zero address");
         _register(subscriptionOrRegistrantToCopy, subscribe);
+        emit LandRegistered(subscriptionOrRegistrantToCopy, subscribe);
```

Comment:
I'd rather put this event into `_register` and make it generic

Reply:
Yes, you are correct; I will change it